src/elarm_event.erl

```diff
@@ -50,19 +50,19 @@
 init(_Opts) ->
     {ok, #evt_state{}}.
 
--spec subscribe(pid()|atom(), sub_filter(), #evt_state{}) ->
-          {{ok,reference()},#evt_state{}}.
+-spec subscribe(pid() | atom(), sub_filter(), #evt_state{}) ->
+          {{ok, reference()}, #evt_state{}}.
 subscribe(Pid, Filter, #evt_state{ subs = Subs } = State)->
     MRef = monitor(process, Pid),
     Ref = make_ref(),
     NewSubs = add_subscriber(Ref, MRef, Pid, Filter, Subs),
     {{ok, Ref}, State#evt_state{ subs = NewSubs }}.
 
 %% Cancel subscription.
--spec unsubscribe(reference(), #evt_state{}) -> {ok,#evt_state{}}.
+-spec unsubscribe(reference(), #evt_state{}) -> {ok, #evt_state{}}.
```

Comment:
The spec in line 62 uses a record, please define a type for the record and use that instead.